src/components/05_pages/NodeForm/index.js

```diff
@@ -74,6 +78,7 @@ class NodeForm extends React.Component {
       entityTypeId: this.props.entityTypeId,
       bundle: this.props.bundle,
     });
+    this.props.requestContentTypes();
```

Comment:
I was thinking that this should probably happen in the `NodeEdit` or `NodeAdd` forms and just be passed as a prop if needed.

Reply:
Hi @lauriii  @mattgrill 
ContentTypes are also missing from the NodeEdit also and NodeForm will be used in case of any content creation and edition so I think we should fetch the contentTypes there.

Doing it inside the actions would require changes at two place - `addContent` and `saveContent` 

What are your thoughts on it?

Reply:
@vishalchandna1 please review the changes I made 🙂 

Agreed that it is a downside that now we have to change two places, but this will ensure that if these actions are triggered anywhere else, that the content types will be requested correctly.

Reply:
ok @lauriii , The changes look cool to me 👍